pkg/kubevirt/core/util.go

```diff
@@ -163,26 +163,47 @@ ethernets:
 	return interfaces, networks, networkData
 }
 
+func findDiskByVolumeName(name string, disks []kubevirtv1.Disk) *kubevirtv1.Disk {
```

Comment:
Could you please move the 2 helper functions `findDiskByName` and `buildDefaultDisk` below `buildVolumes` for consistency with the rest of the code?